e2e/system/build_from_git_test.go

```diff
@@ -1,32 +1,41 @@
 package system
 
 import (
+	"bytes"
 	"context"
 	"strings"
 
 	"github.com/celestiaorg/knuu/pkg/builder"
 )
 
+const (
+	gitRepo   = "https://github.com/celestiaorg/knuu.git"
+	gitBranch = "test/build-from-git" // This branch has a Dockerfile and is protected as to not be deleted
+)
+
 func (s *Suite) TestBuildFromGit() {
 	const namePrefix = "build-from-git"
 	s.T().Parallel()
 
 	// Setup
 	ctx := context.Background()
 
+	s.T().Log("Creating new instance")
 	target, err := s.Knuu.NewInstance(namePrefix)
-	s.Require().NoError(err)
+	if err != nil {
+		s.Require().NoError(err, "Error creating new instance")
+	}
```

Comment:
the if statement is redundant. 

```suggestion
	s.Require().NoError(err, "Error creating new instance")
```

Reply:
not resolved

Reply:
This is outdated, it is already fixed.